event-handlers/src/main/java/no/sikt/nva/nvi/events/UpsertNviCandidateHandler.java

```diff
@@ -60,8 +67,43 @@ private static void validateRequiredCandidateDetails(CandidateDetails candidateD
         Objects.requireNonNull(candidateDetails.verifiedCreators());
     }
 
+    private static DbCandidate toPendingCandidate(CandidateEvaluatedMessage message) {
+        return DbCandidate.builder()
+                   .publicationBucketUri(message.publicationBucketUri())
+                   .publicationId(message.candidateDetails().publicationId())
+                   .applicable(isNviCandidate(message))
+                   .creators(mapToVerifiedCreators(message.candidateDetails().verifiedCreators()))
+                   .level(DbLevel.parse(message.candidateDetails().level()))
+                   .instanceType(message.candidateDetails().instanceType())
+                   .publicationDate(mapToPublicationDate(message.candidateDetails()
+                                                             .publicationDate()))
+                   .points(mapToInstitutionPoints(message.institutionPoints()))
+                   .build();
+    }
+
+    private static List<DbInstitutionPoints> mapToInstitutionPoints(Map<URI, BigDecimal> institutionPoints) {
+        return institutionPoints.entrySet().stream()
+                   .map(entry -> new DbInstitutionPoints(entry.getKey(), entry.getValue())).toList();
+    }
+
+    private static DbPublicationDate mapToPublicationDate(CandidateDetails.PublicationDate publicationDate) {
+        return new DbPublicationDate(publicationDate.year(), publicationDate.month(), publicationDate.day());
+    }
+
+    private static List<DbCreator> mapToVerifiedCreators(List<CandidateDetails.Creator> creators) {
+        return creators.stream()
+                   .map(
+                       verifiedCreatorDto -> new DbCreator(verifiedCreatorDto.id(),
+                                                           verifiedCreatorDto.nviInstitutions()))
+                   .toList();
+    }
+
+    private static boolean isNviCandidate(CandidateEvaluatedMessage evaluatedCandidate) {
```

Comment:
```
private static boolean isNviCandidate(CandidateStatus status) {
        return CandidateStatus.CANDIDATE.equals(status);
    }
```

Reply:
I can flip to putt the Enum first, but == is prettier :)